### PR TITLE
Prepare v0.19.0 release

### PR DESCRIPTION
**Pull request by joshuagl**
Prepare a release to integrate recent changes and update our pinned dependencies in light of a CVE in pyca/cryptography

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## securesystemslib v0.19.0
+
+### Added
+* Enable setting which GPG client to use through an environment variable (#315)
+
+### Changed
+* Dropped support for EOL Python 3.5 and 3.9 (#314)
```
Comment (MVrachev) [inline CHANGELOG.md:9]: The way it's phrased here sounds like we are dropping support for python 3.9 :D

Comment (joshuagl) [reply CHANGELOG.md:9]: Good catch! Let me fix that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+### Added
+* Enable setting which GPG client to use through an environment variable (#315)
+
+### Changed
+* Dropped support for EOL Python 3.5 and add support for Python 3.9 (#314)
+* Converted the default local storage backend, FilesystemBackend, to be a
+  singleton (#302)
+* Migrated CI from travis-ci.org to travis-ci.com (#303) then later to GitHub
+  Actions (#324)
+* Bump dependencies: cffi (#306, 329), cryptography (#322, #333)
```
Comment (jku) [inline CHANGELOG.md:14]:
```suggestion
* Bump dependencies: cffi (#306, 329), cryptography (#322, #333). Latest version of cryptography is no longer used on Python 2 as that is not supported
```

**Review by jku — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (joshuagl) [reply CHANGELOG.md]: This is a nice addition, makes sense to be explicit. Thanks.

**Pull request merged**